R/lint.R

```diff
@@ -287,7 +287,8 @@ lint_package <- function(path = ".", relative_path = TRUE, ...,
 
 
 has_description <- function(path) {
-  file.exists(file.path(path, "DESCRIPTION"))
+  desc_info <- file.info(file.path(path , "DESCRIPTION"))
+  !is.na(desc_info$size) && desc_info$size > 0 && !desc_info$isdir
```

Comment:
@AshesITR any idea why we would get this same lint 3 times?

Reply:
Those three are from three different check runs.
I'd have expected two (one for lint changed files with `.lintr_new` and one for lint package with `.lintr`)

Reply:
Running the same lint 3 times was my doing. It should have run only twice (as @AshesITR correctly indicated - once with `.lintr` and once with `.lintr_new`). One extra run was residual from my early testing when I was forcing a workflow to run on every push (to any branch). I've now removed that since it's excessive.